hackage-server.cabal

```diff
@@ -111,8 +111,8 @@ common defaults
     , process                >= 1.6   && < 1.7
     , text                  ^>= 1.2.5.0 || ^>= 2.0
     , time                   >= 1.9   && < 1.13
-    , transformers           >= 0.5   && < 0.7
-    , unix                   >= 2.7   && < 2.9
+    , transformers           >= 0.5   && < 0.6
+    , unix                   >= 2.7   && < 2.8
```

Comment:
Please don't include untested major versions bump.

Reply:
`transformers` 0.5 is tested while 0.6 is not, and so is `unix`. So I tightened the version bounds. What's the problem?

Reply:
I did it because you mentioned it [here](https://github.com/haskell/hackage-server/commit/f730854464a19a179a7a9300756f57676510e343#r82008276). Did I misunderstand what you mean?

Reply:
Sorry, I confused this (I wasn't expecting these bounds already on master).  All is well.